src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -402,7 +402,7 @@ PerfAnalyzer::WriteReport()
           params_->filename, params_->targeting_concurrency(), summary_,
           params_->verbose_csv, profiler_->IncludeServerStats(),
           params_->percentile, parser_, &writer,
-          params_->should_collect_metrics),
+          params_->should_collect_metrics && params_->verbose_csv),
```

Comment:
let's not have logic inside of the constructor call. Pull it out into a variable and then pass it in

Reply:
Done 👍